autogaita/autogaita_simi_gui.py

```diff
@@ -73,39 +114,9 @@ def donewindow(results, root, root_dimensions):
     # ==> the results dict is not global and will be passed from function to function
     #     like proper programmers do it (nice)
     global cfg  # global cfg variable
-    cfg = {}
-    cfg["analyse_singlerun"] = tk.BooleanVar(root, False)
-    cfg["sampling_rate"] = tk.StringVar(root, "")
-    cfg["dont_show_plots"] = tk.BooleanVar(root, False)
-    cfg["y_acceleration"] = tk.BooleanVar(root, True)
-    cfg["angular_acceleration"] = tk.BooleanVar(root, True)
-    cfg["bin_num"] = tk.StringVar(root, "25")
-    cfg["plot_joint_number"] = tk.StringVar(root, "3")
-    cfg["plot_SE"] = tk.BooleanVar(root, False)
-    cfg["normalise_height_at_SC_level"] = tk.BooleanVar(root, True)
-    cfg["results_dir"] = tk.StringVar(root, "")
-    results = {}  # local results variable
-    results["name"] = tk.StringVar(root, "")
-    results["root_dir"] = tk.StringVar(root, "")
-    results["sctable_filename"] = tk.StringVar(root, "")
-    results["postname_flag"] = tk.BooleanVar(root, False)
-    results["postname_string"] = tk.StringVar(root, "")
-    # joint columns
-    default_joints = ["Midfoot", "Ankle ", "Knee ", "Hip ", "Pelvis "]
-    cfg["joints"] = []
-    for joint in default_joints:
-        cfg["joints"].append(tk.StringVar(root, joint))
-    # angles
-    cfg["angles"] = {}
-    dict_of_defaults = {
-        "name": ["Ankle", "Knee", "Hip"],
-        "lower_joint": ["Midfoot", "Ankle", "Knee"],
-        "upper_joint": ["Knee", "Hip", "Pelvis"],
-    }
-    for key in dict_of_defaults:
-        cfg["angles"][key] = []
-        for joint in dict_of_defaults[key]:
-            cfg["angles"][key].append(tk.StringVar(root, joint))
+    global results
```

Comment:
Is there a specific reason for results to be global? If not, can we keep it local please?

Reply:
I assumed it would be necessary but it doesn't appear to be, so I removed it